airflow/models/dag.py

```diff
@@ -3204,6 +3204,17 @@ def __init__(self, concurrency=None, **kwargs):
     def __repr__(self):
         return f"<DAG: {self.dag_id}>"
 
+    @property
+    def schedule_description(self) -> str | None:
+        if (
+            isinstance(self.schedule_interval, str)
+            and self.schedule_interval != "@once"
+            and self.timetable_description != DatasetTriggeredTimetable.description
+        ):
+            return str(self.schedule_interval) + " " + str(self.timezone.name)
```

Comment:
This weird isinstance check makes me feel the entire logic should go into the timetable class, and we can continue to use `schedule_interval`.